internal/pipeline/dispatcher_test.go

```
@@ -0,0 +1,144 @@
+}
+
+var _ = Describe("Dispatcher", func() {
+	var (
+		ctx    context.Context
+		cancel context.CancelFunc
+		input  chan entity.Message
+		errs   chan entity.PipelineError
+	)
+
+	BeforeEach(func() {
+		ctx, cancel = context.WithCancel(context.Background())
```

golangci-lint (2.12.2)

[error] nested context in function literal

(fatcontext)